### ISSUE TITLE
Possibly wrong Create bindings

### ISSUE THREAD

**Issue by VitalyAtFlare** [fable-compiler/fable-browser#145]
Hi, 

When trying to use the CustomEvent type, I've noticed that the `Create` method doesn't seem to pass its parameters correctly in the generated code. This snippet ([REPL link](https://fable.io/repl/#?code=PYBwpgdgBAQgTsA7gZzHAUOgwgV2QF2AFsBRAN0nwDos4wBDfMKAIhAQHM7lktiRgESgCUGAEwCeLdAB8AfFEQBLCGKRUxS5CEYBjABblKshUo4RgddEA&html=DwQgJg9gxgLgngBwKYAIAWMC2AbAfAKGAxwMLSQEMwCUVgYBLGbJXAUQA8olsUBmAGwBWFAwB2KChLQQArgCdgAekbNW+WsExIYFdDBgIAtEgCOshgDcAvAHIAwhDEwkzowBVESWyihOXznYuHDBKxNgA3L5oFPIAzjrWsjAAZkYAHLY0dNq6KGIU2tYARJYMSADuCBDyMMW+-q4wJRUMYDBo1mBIZdxGre1oADSiYkwMFNhGcVCTSNYAjMXZwDPyDAgwKHHyUCUYhnEAXEpKUGBiAHQAVnHd2Fbyl2I6SmIImEpltbKTRpCYAACACZLgswUowAw4qFvjBflMATc4vUoPIIHE4jUGABzcS4ZRrDYwFYwuAsbK0ABGEDAcBQAG8NLQWQgqFCxDijighAAGBAcCLMlkoFL+IwpQoMbBwblxKRxaZIdYpIUigC++GFKF0VJYjO11Jq3Xk0zZUHEXJQ-MFhpQNPkJqMNIMEEw3IWAu2EAeYBQAGIkLyg0G1SL7cblUZ1jiMB6vVjfQGQ8HeWHaJrtTAwEMs2gDeGUAMOty+QL0yyHU6WCkYPGON6k4HU6G7VWozAIAh6422smW2m7WywByrUJy3aIJZlSlsBAKty0G1umIK7RgjAjFOZ3OFygeA8EHFoRXMyLs5cEixYEg-UzC8PR9ybaetee-eIEMkC+Hi2hubM2BQAAFAsvK8gApCgRg8gKACUa7oEguJxigwLAhOhaYLEeJiM+mEspqmhKGSFKEGElDUFqwA0nSKxQpYohgCU2HiMsygMQQyi0XApDKOEBBAA&css=Q), but also confirmed in a small local compiled project):

```FSharp
open Browser

CustomEvent.Create "progressComponentReady"
|> window.dispatchEvent
|> ignore
```

Compiles to this code:
```JS
(() => {
    const evt = new CustomEvent()("progressComponentReady");
    return window.dispatchEvent(evt);
})();
```

As you can see, it passes no parameters to CustomEvent constructor, but then applies the parameter string to the object itself.

**Comment by MangelMaxime** [comment]
Hello @VitalyAtFlare,

It seems like this issue has been fixed in [`1.4.6`](https://github.com/fable-compiler/fable-browser/blob/master/src/Event/RELEASE_NOTES.md#146) and looking at the code it seems to be ok to me.

I confirm that Fable REPL still generate invalid code but this is probably because we didn't update the dependency metadata their. I am making a note to update them to be sure that we use the latest versions

**Comment by VitalyAtFlare** [comment]
Thanks for the quick reply @MangelMaxime! Can confirm, local testing with the latest version generates correct code.

Closing.
